store/sqlstore/channel_store.go

```diff
@@ -4255,3 +4256,51 @@ func (s SqlChannelStore) GetTopChannelsForUserSince(userID string, teamID string
 
 	return model.GetTopChannelListWithPagination(channels, limit), nil
 }
+
+func (s SqlChannelStore) PostCountsByDay(channelIDs []string, sinceUnixMillis int64) ([]*model.DailyPostCount, error) {
+	dailyPostCounts := make([]*model.DailyPostCount, 0)
+
+	postgresPropQuery := `AND (Posts.Props ->> 'from_bot' IS NULL OR Posts.Props ->> 'from_bot' = 'false')`
+	mySqlPropsQuery := `AND (JSON_EXTRACT(Posts.Props, '$.from_bot') IS NULL OR JSON_EXTRACT(Posts.Props, '$.from_bot') = 'false')`
```

Comment:
And for identifying bot posts, the login on the frontend also takes into account whether or not the poster is a bot user which could complicate this. That logic also seems backwards to what I'd expect though since it only seems to show the BOT tag if the user isn't a bot ([link](https://github.com/mattermost/mattermost-webapp/blob/master/components/post_view/post_header/post_header.tsx#L123)).

It might not be worth taking that into account then since the logic is kind of odd, and hopefully this wouldn't have a big effect on the results in most cases